gef.py

```diff
@@ -503,22 +503,22 @@ def wrapper(*args: Any, **kwargs: Any) -> Callable:
 class Color:
     """Used to colorify terminal output."""
     colors = {
-        "normal"         : "\033[0m",
-        "gray"           : "\033[1;38;5;240m",
-        "light_gray"     : "\033[0;37m",
-        "red"            : "\033[31m",
-        "green"          : "\033[32m",
-        "yellow"         : "\033[33m",
-        "blue"           : "\033[34m",
-        "pink"           : "\033[35m",
-        "cyan"           : "\033[36m",
-        "bold"           : "\033[1m",
-        "underline"      : "\033[4m",
-        "underline_off"  : "\033[24m",
-        "highlight"      : "\033[3m",
-        "highlight_off"  : "\033[23m",
-        "blink"          : "\033[5m",
-        "blink_off"      : "\033[25m",
+        "normal"         : "\001\033[0m\002",
```

Comment:
Let's add a comment explaining the codes (with a link).

Reply:
Also, why do we need these? Doesn't the `\033[0m` end the sequence? I think we just need to make sure to use `normal` at the end whenever we are done using a colour.